src/hotspot/cpu/x86/nativeInst_x86.cpp

```diff
@@ -510,12 +510,28 @@ void NativeJump::check_verified_entry_alignment(address entry, address verified_
 //
 void NativeJump::patch_verified_entry(address entry, address verified_entry, address dest) {
   // complete jump instruction (to be inserted) is in code_buffer;
+#ifdef AMD64
```

Comment:
Just a minor suggestion: `_LP64` is more appropriate here since it's x86-specific file.

Reply:
done